docs/source/examples/checks/vision/performance/confusion_martix.ipynb

```diff
@@ -0,0 +1,9265 @@
+{
```

Comment:
<p>Can't see anything still :( Maybe the problem is in my browser, will check later</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/828/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>